packages/block-editor/src/components/inserter/menu.js

```diff
@@ -101,12 +101,14 @@ function InserterMenu(
 			window.requestAnimationFrame( () => {
 				if (
 					! shouldFocusBlock &&
-					! blockTypesTabRef?.current.contains(
+					! blockTypesTabRef?.current?.contains(
 						ref.current.ownerDocument.activeElement
 					)
 				) {
 					// There has been a focus loss, so focus the first button in the block types tab
-					blockTypesTabRef?.current.querySelector( 'button' ).focus();
+					blockTypesTabRef?.current
```

Comment:
Same as above

```suggestion
					blockTypesTabRef.current
```

Reply:
Cleaned up in 112547b11005ba1eaabb64d54bfea4379f8c3e33